### ISSUE TITLE
verbose

### ISSUE THREAD

**Issue by maxwell1386** [RL-VIG/LibFewShot#42]
Thanks for sharing this code. 
Is there any option to see the progress of training or live log display during training?

**Comment by yangcedrus** [comment]
Thanks for your issue.
We now have train/val/test acc, data time, cal time and loss shown during training. We are not sure what progress you mean, do you mean the curve like Tensorboard or some other form?  
If you are referring to tensorboard, we also save the tensorboard file in the 'results' directory, which can be viewed using the tensorboard command.

**Comment by maxwell1386** [comment]
You are welcome.
I mean how can I see train/val/test acc, data time, cal time and loss during training or how can I active verbose.
During training I can not see these parameters.and I have to open related "log" files of them in result folder to see them. is there any option to activate it?

**Comment by yangcedrus** [comment]
Hello, we have not met this problem you mentioned yet.
We are wondering if you could check or tell us the following points:
1. Have any modifications been made to the source code or configuration file?
2. Check the information of OS and Python version.
3. Check the environment variables.

**Comment by maxwell1386** [comment]
Thank you.
I am running the code on Google Colab.

**Comment by wZuck** [comment]
This problem may caused by some limitation of the Colab(or jupyter), could you try to add a magic `%%python` before run your cell?
```python
%% python
import sys
sys.dont_write_bytecode = True
from core.config import Config
from core import Trainer

if __name__ == "__main__":
    config = Config("./config/proto.yaml").get_config_dict()
    trainer = Trainer(config)
    trainer.train_loop()

```

Generally, the LibFewShot should be run via shell commands, could you try run the `run_trainer.py` by following command in the Colab?
```python
!python3 run_trainer.py
```

**Comment by maxwell1386** [comment]
Thank you so much.
It works good via shell.
